wsgi.py

```diff
@@ -1,19 +1,17 @@
 import os
-import multiprocessing
 from gunicorn.app.base import BaseApplication
 from app import app as application
 
 
-class StandaloneApplication(BaseApplication):
+class StandaloneApplication(BaseApplication):  # pylint: disable=abstract-method
```

Comment:
@ltkum, @procrastinatio here also we may globally disable the redefined-outer-name warning ? For a C++ programmer point of view I would not do it, but I see as a Python programmer that this code idiom could be handy. If this is often used I would then disable it globally to avoid too many pylint comment.